website/docs/understanding_ferretdb.md

```diff
@@ -82,8 +82,8 @@ To reference the fourth field in the array, use the dot notation `"animals.3"`.
 
 Here are more examples of dot notations on arrays:
 
-* [Query an array](https://docs.ferretdb.io/Basic_operations/read/#retrieve-documents-containing-a-value-in-an-array)
-* [Update an array](https://docs.ferretdb.io/Basic_operations/update/#update-an-array-element)
+* [Query an array](./basic_operations/read/#retrieve-documents-containing-a-specific-value-in-an-array)
+* [Update an array](./basic_operations/update.md#update-an-array-element)
```

Comment:
Hmm, here we have `.md` in the link text, but not in the other places? Why?

Reply:
Thanks, `.md` should be added.